server/logger.ts

```diff
@@ -41,6 +41,14 @@ const logger = winston.createLogger({
         hformat
       ),
     }),
+    new winston.transports.File({
+      filename: path.join(__dirname, '../config/logs/tmp.log'),
+      format: winston.format.combine(
+        winston.format.splat(),
+        winston.format.timestamp(),
+        winston.format.json()
+      ),
+    }),
```

Comment:
Would the `tmp.log` file just grow indefinitely?  If so, that would defeat the purpose of logrotate...

I think it's okay for the log files to all be in JSON format, but we do need to rotate them daily and compress/delete old logs.

Reply:
We should create a migration path to JSON logs. On startup, delete the existing daily file if it's not JSON format and then let Winston recreate it.